snappass/main.py

```diff
@@ -22,13 +24,29 @@
 }
 
 
+def check_redis_alive(fn):
+    def inner(*args, **kwargs):
+        try:
+            redis_client.ping()
+        except ConnectionError as e:
+            print(e)
+            if fn.__name__ == "main":
```

Comment:
Minor nit: can we switch this to single quotes? I haven't done it yet, but I am going to write a coding style for this repo, and strings favor single quotes if possible.